src/modules/config.js

```diff
@@ -364,6 +428,9 @@ const Config = () => {
         },
       },
 
+
```

Comment:
unnecessary newlines.  if you're using vscode and install a code formatter extension like `esbenp.prettier-vscode` it will use the .prettierrc in the repo to automatically correct stuff like this and help keep things more consistent.

Reply:
Cleaned up whitespace correctly hopefully (I installed the extension but I don't know if the config is right)